examples/window/window_settings.rs

```diff
@@ -16,6 +16,8 @@ fn main() {
                     title: "I am a window!".into(),
                     resolution: (500., 300.).into(),
                     present_mode: PresentMode::AutoVsync,
+                    // Tells wasm to resize the window according to the available canvas
+                    fit_canvas_to_parent: true,
```

Comment:
```suggestion
                    // Tells wasm to set the CSS width and height properties to 100% for the target canvas.
                    // This can be useful to support window resizing.
                    fit_canvas_to_parent: true,
```

Maybe an improvement ?